apps/mobile/src/components/action-bar.tsx

```diff
@@ -0,0 +1,49 @@
+import type { ReactNode } from 'react';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { useTheme } from '@shopify/restyle';
+import { BlurView } from 'expo-blur';
+
+import { Box, Theme } from '@leather.io/ui/native';
+
+export const ACTION_BAR_HEIGHT = 70;
+export const ACTION_BAR_BOTTOM_OFFSET = 40;
+export const ACTION_BAR_TOTAL_HEIGHT = ACTION_BAR_HEIGHT + ACTION_BAR_BOTTOM_OFFSET;
+
+export function ActionBar(props: { left?: ReactNode; center?: ReactNode; right?: ReactNode }) {
+  const { bottom } = useSafeAreaInsets();
+  const theme = useTheme<Theme>();
+
+  return (
+    <Box width="100%" px="5" justifyContent="center" alignItems="center">
```

Comment:
Can we use design tokens for `px=5` ? Or does it not line up correctly. 

Or else this isn't pixel perfect yet in which case no worries

Reply:
We could try and use the tokens we already have in the `tokens` package?

```
    'space.00': { value: '0' },
    'space.01': { value: '4px', description: '4px' },
    'space.02': { value: '8px', description: '8px' },
    'space.03': { value: '12px', description: '12px' },
    'space.04': { value: '16px', description: '16px' },
    'space.05': { value: '24px', description: '24px' },
    'space.06': { value: '32px', description: '32px' },
    'space.07': { value: '40px', description: '40px' },
```

 They seem almost the same although we would have to convert `px` string to number but I have a function for that in the `utils` package `pxStringToNumber`